cartography/intel/aws/rds.py

```diff
@@ -98,6 +104,33 @@ def load_rds_instances(neo4j_session, data, region, current_aws_account_id, aws_
             AWS_ACCOUNT_ID=current_aws_account_id,
             aws_update_tag=aws_update_tag
         )
+    _attach_read_replicas(neo4j_session, read_replicas, aws_update_tag)
+
+
+def _attach_read_replicas(neo4j_session, read_replicas, aws_update_tag):
+    """
+    Attach read replicas to their source instances
+    """
+    attach_replica_to_source = """
+    MATCH (replica:RDSInstance{id:{ReplicaArn}}),
+    (source:RDSInstance{db_instance_identifier:{SourceInstanceIdentifier}})
+    MERGE (replica)-[r:IS_READ_REPLICA_OF]->(source)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    """
+    for replica in read_replicas:
+        if not replica.get('ReadReplicaSourceDBInstanceIdentifier'):
+            logger.debug(f"Expected RDSInstance to be a read replica but its ReadReplicaSourceDBInstanceIdentifier "
+                         f"field is None.  Here is the object: {replica}")
```

Comment:
Python's logging library handles string formatting in the "old" way and we use that through the rest of the codebase. For consistency's sake please use old-style formatting when logging. See https://github.com/lyft/cartography/blob/master/cartography/sync.py#L65